target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -71,15 +84,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "3:lan:4" "4:lan:3" "5:lan:2" "2:wan"
 		;;
-	comfast,cf-e5|\
```

Comment:
cf-e5 is deleted, but not recreated

Reply:
I meant to do this. It defaults to the last case in this script.

Reply:
Ah, I see. I always thought of the default case as an emergency fallback, but why not use it ...